locales/en/badge.json

```
@@ -1,12 +1,12 @@
 {
   "title": {
-    "connectWallet": "You must connect wallet first",
-    "mintable": "You can mint this badge with",
-    "notMintable": "You canont mint this badge. to Get this badge you need to met certain criteria",
-    "freeMintable": "You can mint this badge free",
-    "minted": "🎉🎉🎉 You have Badge already !! "
+    "connectWallet": "You must connect to a wallet first!",
+    "mintable": "You can mint this Badge with",
```

Why is "Badge" capitalised? It looks like a regular noun to me.

(Even if we were branding Hekaku Badge or anything like that, I wouldn't capitalise "Badge" alone because the brand would not be well-known enough at this point.)

Suggestion:

```suggestion
    "mintable": "You can mint this badge with",
```

Note: there are more occurrences of "Badge".